torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -3136,6 +3136,16 @@ struct to_ir {
         return emitBuiltinCall(
             tree->range(), *method.graph(), kind, named_values, {});
       }
+      case '%': {
+        if (tree->tree(0)->kind() == TK_STRINGLITERAL) {
```

Comment:
Could you give me an example test case? I'm looking at possible tokens in `lexer.h`, and I don't see what else it could be. (`TK_STRING` is reserved for atoms.)

Reply:
I was thinking about something like:

def fn(x: str):
    return x % "hi"